src/test/java/org/apache/commons/io/build/AbstractOriginTest.java

```diff
@@ -130,4 +133,12 @@ public void testGetWriter() throws IOException {
     public void testSize() throws IOException {
         assertEquals(Files.size(Paths.get(FILE_NAME_RO)), getOriginRo().getByteArray().length);
     }
+
+    @Test
+    public void testURI() throws Exception {
```

Comment:
This test is in the wrong class, it does not do anything generic that would apply to all of its subclasses. Use `URIOriginTest`

Reply:
Yes, I updated.